test/system/200-pod.bats

```diff
@@ -478,26 +483,36 @@ spec:
         skip "only meaningful for rootful"
     fi
 
+    mkdir /dev/foo
+    mknod /dev/foo/null b 1 3
```

Comment:
No no no. This is vandalism: not only does it corrupt an existing system, it prevents the test from ever running again because there's no cleanup (and even if there was cleanup, it would not run if there was an error). We do not vandalize our test system.

Reply:
ok @edsantiago thanks for the feedback. I always forget that system tests have these consequences. I will mimic something more like the loopback device in 180-blkio.bats